keepassxc-browser/_locales/en/messages.json

```diff
@@ -599,6 +599,10 @@
         "message": "Activate autocomplete for username fields.",
         "description": "Activate autocomplete for username fields checkbox text."
     },
+    "optionsCheckboxShowGroupNameInAutocomplete": {
+        "message": "Display the group name of the credentials in the autocomplete list",
```

Comment:
Recommend this text: "Display the group name in autocomplete list when credentials are from different groups"